mod/notifications/start.php

```diff
@@ -113,6 +114,63 @@ function _notifications_page_menu($hook, $type, $return, $params) {
 }
 
 /**
+ * Register menu items for the page menu on group profiles
+ *
+ * @param \Elgg\Hook $hook hook
+ *
+ * @return array
+ *
+ * @access private
+ *
+ * @since 3.0
+ */
+function _notifications_groups_subscription_page_menu(\Elgg\Hook $hook) {
```

Comment:
i just moved it, but i think the idea was to have a status indication on your notification preference on/off. I am fine with just changing this to a simple link. Anybody else with an opinion?